build-docker-image/action.yml

```diff
@@ -16,6 +16,9 @@ inputs:
   image_name:
     description: Docker image name WITHOUT registry and WITHOUT Docker tag, e.g. example/image.
     required: true
+  build_args:
+    description: Docker build args to append to `docker build` command.
+    required: false
 
 runs:
   using: composite
```

Comment:
For debugging purposes it would also be nice to show the new input with `echo` in the `"Echo inputs"` step please

Reply:
Added this.